src/prefect/server/api/deployments.py

```diff
@@ -558,7 +560,7 @@ async def set_schedule_inactive(
                 status_code=status.HTTP_404_NOT_FOUND, detail="Deployment not found"
             )
         deployment.is_schedule_active = False
-        deployment.paused = False
+        deployment.paused = True
```

Comment:
@bunchesofdonald can you help me make sure this is okay?  I wanted to match Prefect Cloud, and this is the only implementation drift I saw